### ISSUE TITLE
[19125] Allow portal to add CC to ticket

### ISSUE THREAD

**Issue by Nness** [DeskDirector/UserVoice#346]
Allow ticket owner to add other users into ticket, to be referenced.

**Comment by dr3dgen** [comment]
Thank you for raising this issue, @Nness. We've found this to be a bit of a barrier for tickets that involve more than one person - primarily for office managers/owners/HR/SMEs when making purchase orders, creating workstation builds for new employees, and workstation shutdowns for employee exits. 

Clients can't CC any appropriate teammate (and we can't CC them from DD Tech). When added in ConnectWise by our team, CC clients receive emails from ConnectWise, but the CC'd ticket does not appear anywhere in their personal portal. This has caused some communication/workflow challenges with getting ticket communication out of email.

At one site, we have a user who is the primary contact and can only access the tickets in DD Portal by 1) searching for the email, 2) copying the ticket number, 3) going into DD Portal under open tickets, 4) clicking Everyone's Tickets, 5) pasting the ticket number to find, and 6) adding a note accordingly. Each time there is a change or update in the ticket, this process has to be repeated. 

Once CC'd, another major issue is there are no notifications in DD Portal that the user has been added as a contact. If it showed up in Notifications and redirected to the appropriate ticket as a workaround, I think at very least that would be a start. 

I believe this feature is desperately needed. Chat is an evolution of some of these features, but the saved chats are not as visible/readily queryable in the ticket without digging to see what happened, and attachments across people are not readily accessible through chat.

**Comment by Lowell-Syncrety** [comment]
Yes, I'm finding that I abandon using DD Tech the first time I need to communicate something back to the ticket contact or someone else who is (or needs to be) CC'd on the ticket. It should also be made more obvious when the ticket contact will actually get an email from the ticket note. Right now it doesn't say they will ever get an email when you write a note.

**Comment by awiedeman** [comment]
I really REALLY don't think you need to put security around this - so you should scrap that consideration.  I say this because anyone can reply to an email from Manage and CC ANYONE and they would automatically get added to the ticket in Manage (based off of your Manage setting of course).  

There is literally no security for this in Manage, why should there be in Deskdirector??  It's either ON or OFF.  If it's ON in Manage, it should be ON in DD and vice versa!

**Comment by Andrew-Lahikainen** [comment]
For context I think @awiedeman is referring to this comment made by me in a support ticket

> We're getting more requests for this so it's definitely top of mind for us at the moment, and it definitely makes sense for us to implement. We need to think about the design around this, mostly around the permissions of the CCs and how they can view the tickets they don't own, but are CCs on.

I was talking about permissions and how this feature would necessarily interact with current permissions (Approvers, All Tickets, Default Contact, etc). We also need to be mindful that ConnectWIse is not the only PSA we support, we need to see what the differences are between Autotask and ConnectWise and see if there is a way we can abstract the feature to work for both.

**Comment by Aycorn** [comment]
Similar Request: Service Ticket #21263 - Can we add a Cc email address

**Comment by awiedeman** [comment]
For now, can we add the ability to at least SEE the CCs in the ticket? Thats what my clients are asking for...

**Comment by awiedeman** [comment]
Hi there - any update on this? Thanks

**Comment by Andrew-Lahikainen** [comment]
@awiedeman we're going to be adding a field so users can see CCs this week

**Comment by iskolares** [comment]
Related DD ticket #25114

**Comment by iskolares** [comment]
Related request DD ticket #25133
